examples/calculator_server_shttp_sse.ts

```diff
@@ -0,0 +1,173 @@
+import express from "express";
```

Comment:
Added this file for manual testing & for use via the added example because we don't yet have an integration suite.